<launch>
  <!-- <node type="rviz" name="rviz" pkg="rviz" args="-d $(find kf_applied)/rviz/kf_applied.rviz" /> -->

  <node name="kf_applied_node" pkg="kf_applied" type="kf_applied_node" respawn="false" output="screen">
    <!-- launch-prefix="gdb -ex run"> -->

    <!-- general -->
    <rosparam param="measurements">[gt, gps, radar]</rosparam>

    <!-- bag topics -->
    <param name="frame_id" type="string" value="global"/>
    <param name="topic_gt" type="string" value="/kf_applied/gt"/>
    <param name="topic_gps" type="string" value="/kf_applied/gps"/>
    <param name="topic_radar" type="string" value="/kf_applied/radar"/>
    <param name="topic_estimation" type="string" value="/kf_applied/estimation"/>

    <!-- bag parameters -->
    <param name="path_bag" type="string" value="/home/alericcardi/Workspaces/ws_kf/src/kalman_filter_applied/data/1611219708.bag"/>

    <!-- sensor noise values / update -->
    <rosparam param="gps_noise">[0.15, 0.15, 0.05]</rosparam>

  </node>



</launch>